.azure/gpu-tests.yml

```diff
@@ -95,6 +95,18 @@ jobs:
               --durations=250 \
               --timeout=240 \
               --numprocesses=9 \
+              -m "not large_mem" \
+              --ignore=thunder/tests/distributed --ignore=thunder/tests/test_networks.py
+          coverage run --source thunder -m \
+            pytest thunder/tests/ \
+              -m "not standalone" \
+              -v --datefmt="%Y%m%d-%H:%M:%S.%f" \
+              --timeout=240 \
+              --random-order-seed=42 \
+              --durations=250 \
+              --timeout=240 \
+              --numprocesses=4 \
```

Comment:
[question] what's the reasoning/heuristic to set this to 4?

Reply:
half of 9

Reply:
and the rest 0.5 is just tax :)